core/src/cats/ddl/creates/postgresql.sql

```diff
@@ -496,28 +490,28 @@ SELECT
   f.fileset AS fileset,
   j.jobbytes / 1000000 AS total_mb,
   CASE
-    WHEN f.filesettext ILIKE '%{%{%File%=%' THEN NULL
+    WHEN f.filesettext ILIKE '%{%{%File%=%' AND f.filesettext NOT ILIKE '%:config_file%=%' THEN NULL
```

Comment:
Ignoring `config_file=` feels oddly specific.
Wouldn't reordering the `WHEN`s to move the `File=` to the bottom achieve the same goal?
Otherwise we could rewrite `ILIKE '%{%{%File%=%'` to something like `SIMILAR TO '%{%{\s*File\s*=%'` which is less portable, but as we're PostgreSQL-only now shouldn't be an issue.

Reply:
I would avoid regexp but in that particular case, it is certainly the best possible option as it will highly simplifies the string especially if in the same time we move the file to the end of each block.
Will do a proposal with that solution.